legateboost/models/tree.py

```diff
@@ -32,8 +33,15 @@ class Tree(BaseModel):
     split_samples : int
         The number of data points to sample for each split decision.
         Max value is 2048 due to constraints on shared memory in GPU kernels.
-    alpha : float
-        The L2 regularization parameter.
+    l1_regularization : float
+        The L1 regularization parameter applied to leaf weights.
+    l2 : float
+        The L2 regularization parameter applied to leaf weights.
+    alpha : deprecated
+        Deprecated, use `l2_regularization` instead.
+    min_split_gain : float
+        The minimum improvement in the loss function required to make a split.
+        Increasing this value generates smaller trees.
```

Comment:
As discussed, may make sense to note that this is an improvement per number of classes.